### ISSUE TITLE
Docs state to update as follows, but does not mention what to do

### ISSUE THREAD

**Issue by ShubhankarKG** [angular/angular#40280]
# 📚 Docs or angular.io bug report

### Description

 
On the page [Adding Navigation](https://angular.io/start/start-routing), there is a section where we need to update the products by adding a productId. The statement reads, and I quote:
> Update the *ngFor directive to read as follows. This statement instructs Angular to iterate over the items in the products array and assigns each index in the array to the productId variable when iterating over the list.

But there is nothing below that actually does this. Was this intentional? If it was, maybe rephrase the sentence to mean that **you** need to do it.

### What's the affected URL?**
 
https://angular.io/start/start-routing

### Expected vs Actual Behavior**

I expected a code stub that succeded this statement, indicating that **this** is what you need to do, as a newbie :sweat_smile: .

## 📷Screenshot

![Screenshot from 2020-12-30 19-15-01](https://user-images.githubusercontent.com/50791000/103355076-5afcbd80-4ad3-11eb-9cfc-ce318cb3f331.png)

### Anything else relevant?

If this is an easy fix, maybe I myself can try this out. Thanks.

**Comment by sonukapoor** [comment]
@ShubhankarKG The steps 3-5 are all referring to the code block that you see after step 5.

**Comment by ShubhankarKG** [comment]
Oh, thanks. Then the statement : 
>and assigns each index in the array to the productId variable when iterating over the list. 

is misleading, don't you think so? Since the previous tutorial did not have the `id` attribute at all.

**Comment by sonukapoor** [comment]
Yes. That is misleading, as there is no usage of the `productId` in the code section.

**Comment by ShubhankarKG** [comment]
Thanks! Then I guess that we can add that to the starter code itself. That'll eliminate this inconsistency I think.

If it's not too much of a bother, could you let me know where this code resides so that I can make a quick fix? 

Thanks!

**Comment by sonukapoor** [comment]
Sure, it's right here:

https://github.com/angular/angular/blob/master/aio/content/start/start-routing.md

**Comment by gkalpak** [comment]
Thx for reporting this, @ShubhankarKG!
I am going to close this as a duplicate of #40189. (It will be fixed by #40197.)
